Formula/c/clinfo.rb

```diff
@@ -32,6 +32,12 @@ def install
   end
 
   test do
-    assert_match(/Device Type +[CG]PU/, shell_output(bin/"clinfo"))
+    output = shell_output(bin/"clinfo")
+    # OpenCL does not work on ephemeral ARM CI.
+    if Hardware::CPU.arm? && OS.mac? && Hardware.virtualized? && ENV["HOMEBREW_GITHUB_ACTIONS"]
+      assert_match("number of devices : error -30", output, 1)
```

Comment:
The `1` should be an argument to `shell_output`, not `assert_match` 😄 

```ruby
output = shell_output(bin/"clinfo", 1)
[...]
assert_match("number of devices : error -30", output)
```

Reply:
🤦‍♂️ Thank you. Fixed.